### DISCUSSION TITLE
onTeamId not working?

### DISCUSSION THREAD

**Discussion by k3ls3yP** [cwendt94/espn-api#386] [ESPN API]
Hello! I keep running into an issue where the player no longer has the onTeamId functionality.
My code:
`for player in team_lineups:
            if player_name == player.name:
                searched_player = player

        team_int = searched_player.onTeamId
        fantasy_team = NULL
        if team_int == 0:
            fantasy_team = 'Free Agent'
        else:
            fantasy_team = league.teams[team_int-1].team_name`

The error:
`Traceback (most recent call last):
  File "C:\Users\kelse\AppData\Local\Programs\Python\Python310\lib\site-packages\discord\ext\commands\core.py", line 190, in wrapped
    ret = await coro(*args, **kwargs)
  File "C:\Users\kelse\source\Bots\FootyBot\discordbotclient.py", line 274, in player
    team_int = searched_player.onTeamId
AttributeError: 'int' object has no attribute 'onTeamId'

The above exception was the direct cause of the following exception:

`Traceback (most recent call last):
  File "C:\Users\kelse\AppData\Local\Programs\Python\Python310\lib\site-packages\discord\ext\commands\bot.py", line 1347, in invoke
    await ctx.command.invoke(ctx)
  File "C:\Users\kelse\AppData\Local\Programs\Python\Python310\lib\site-packages\discord\ext\commands\core.py", line 986, in invoke
    await injected(*ctx.args, **ctx.kwargs)  # type: ignore
  File "C:\Users\kelse\AppData\Local\Programs\Python\Python310\lib\site-packages\discord\ext\commands\core.py", line 199, in wrapped
    raise CommandInvokeError(exc) from exc
discord.ext.commands.errors.CommandInvokeError: Command raised an exception: AttributeError: 'int' object has no attribute 'onTeamId'`

I've got the most updated git and it has worked previously and just decided not to today. Any advice?

**Comment by dtcarls** [comment]
I was able to run this without any issue using espn_api 0.23.0
```
from espn_api.football import League
league = League(league_id=league_id, year=2022)
box_scores = league.box_scores()
for i in box_scores:
    for player in i.home_lineup:
        searched_player = player

        team_int = searched_player.onTeamId
        print(team_int)
        fantasy_team = ''
        if team_int == 0:
            fantasy_team = 'Free Agent'
        else:
            fantasy_team = league.teams[team_int-1].team_name

**Comment by cwendt94** [comment]
From this error `AttributeError: 'int' object has no attribute 'onTeamId'` it seems like the `searched_player` field is a int and not a player class object.
